g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/CLDRUtils.java

```diff
@@ -184,8 +193,8 @@ public static String readZip(String fileName, String zipPath) {
         dateMap.put(Constants.MONTH_FORMAT, monthsFormatMap);
         dateMap.put(Constants.MONTHS_STANDALONE, monthsStandaloneMap);
         dateMap.put(Constants.ERAS, erasMap);
-        dateMap.put(Constants.FIRST_DAY_OF_WEEK, 0);
-        dateMap.put(Constants.WEEKEND_RANGE, Arrays.asList(6, 0));
+        dateMap.put(Constants.FIRST_DAY_OF_WEEK, firstDayOfWeek);
+        dateMap.put(Constants.WEEKEND_RANGE, weekendRange);
```

Comment:
Why add weekend range. How does this relate to the first day of week?

Reply:
As weekend range has the same bug, and need similar code change, so I fix them together.

I have asked @nannany00 include weekend range problem in the issue #879, and change PR title accordingly.